### PR TITLE
add shuffle

### PR DESCRIPTION
**Pull request by WenheLI**

### CODE REVIEW

File: test/node/dataset/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { DatasetMeta, DatasetSize, DatasetType, makeDataset, Sample } from '../../../src/dataset';
+import {  makeDataset, types } from '../../../src/dataset';
```
Comment (FeelyChau) [inline test/node/dataset/index.ts:2]:
```suggestion
import { makeDataset, types } from '../../../src/dataset';
```

**Review by FeelyChau — CHANGES_REQUESTED** [review]

**Comment by FeelyChau** [comment]
Could we enable the unit test in this PR?

**Comment by WenheLI** [comment]
> Could we enable the unit test in this PR?

Unit test is enabled under `npm install`. We should fix it in a separate PR.

**Review by FeelyChau — APPROVED** [review]
LGTM.

**Pull request merged**